uploader.json

```diff
@@ -1167,12 +1290,16 @@
         "block": "/bin/ls /sys/block | awk '!/^ram|^\\.+$/ {print \"/dev/\" $1 \" unit s print\"}'",
         "ceph_osd_ec_profile_ls ": "/usr/bin/ceph osd erasure-code-profile ls",
         "ceph_socket_files": "/bin/ls /var/run/ceph/ceph-*.*.asok",
+        "docker_containers": "/usr/bin/docker ps -aq",
+        "docker_images": "/usr/bin/docker images -q",
```

Comment:
This spec returns a massive amount of information on some systems. Why are we doing this?

Reply:
They are `pre_command` needed by `docker_container_inspect` and `docker_image_inspect`

Reply:
Ok. We should keep an eye on the effectiveness of the associated downstream components to be sure the value outweighs the cost as time goes on.